src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -579,4 +610,12 @@ TransportNetwork getTransportNetwork(String idString) {
 	private String getString(int res) {
 		return context.getString(res);
 	}
+
+	private String region(String name, String flag) {
+		return flag + " " + name;
+	}
+
+	private String description(String desc, String networks) {
```

Comment:
Nice idea to have helper methods for that!

You could even improve that further like so: Except for `flag` all those could take a `@StringRes int` type as parameter, so you can remove all the `getString()` methods above and use them here instead.